go.mod

```diff
@@ -55,6 +55,7 @@ require (
 	github.com/StackExchange/wmi v1.2.1 // indirect
 	github.com/antlr4-go/antlr/v4 v4.13.0 // indirect
 	github.com/asaskevich/govalidator v0.0.0-20230301143203-a9d515a09cc2 // indirect
+	github.com/aws/aws-sdk-go v1.50.25 // indirect
```

Comment:
why do we need extra deps? we do the very same logic in the production code, I would expect to reuse the same existing vendored deps